source/common/stats/hystrix.cc

```diff
@@ -59,117 +55,117 @@ void Hystrix::addInfoToStream(std::string key, std::string value, std::stringstr
-  addIntToStream("rollingCountCollapsedRequests",   0,          cluster_info);
-  addIntToStream("rollingCountExceptionsThrown",    0,          cluster_info);
-  addIntToStream("rollingCountFailure",             errors,     cluster_info);
-  addIntToStream("rollingCountFallbackFailure",     0,          cluster_info);
-  addIntToStream("rollingCountFallbackRejection",   0,          cluster_info);
-  addIntToStream("rollingCountFallbackSuccess",     0,          cluster_info);
-  addIntToStream("rollingCountResponsesFromCache",  0,          cluster_info);
+  double error_rate = total == 0 ? 0 : ((errors + timeouts + rejected) / total) * 100;
+
+  addIntToStream("errorPercentage", error_rate, cluster_info);
+  addIntToStream("errorCount", errors, cluster_info);
+  addIntToStream("requestCount", total, cluster_info);
+  addIntToStream("rollingCountCollapsedRequests", 0, cluster_info);
+  addIntToStream("rollingCountExceptionsThrown", 0, cluster_info);
+  addIntToStream("rollingCountFailure", errors, cluster_info);
+  addIntToStream("rollingCountFallbackFailure", 0, cluster_info);
+  addIntToStream("rollingCountFallbackRejection", 0, cluster_info);
+  addIntToStream("rollingCountFallbackSuccess", 0, cluster_info);
+  addIntToStream("rollingCountResponsesFromCache", 0, cluster_info);
 
   // Envoy's "circuit breaker" has similar meaning to hystrix's isolation
   // so we count upstream_rq_pending_overflow and present it as rejected
-  addIntToStream("rollingCountSemaphoreRejected",   rejected,   cluster_info);
+  addIntToStream("rollingCountSemaphoreRejected", rejected, cluster_info);
 
   // Hystrix's short circuit is not similar to Envoy's since it is trrigered by 503 responses
   // there is no parallel counter in Envoy since as a result of errors (outlier detection)
   // requests are not rejected, but rather the node is removed from load balancer healthy pool
-  addIntToStream("rollingCountShortCircuited",      0,          cluster_info);
-  addIntToStream("rollingCountSuccess",             success,    cluster_info);
-  addIntToStream("rollingCountThreadPoolRejected",  0,          cluster_info);
-  addIntToStream("rollingCountTimeout",             timeouts,   cluster_info);
-  addIntToStream("rollingCountBadRequests",         0,          cluster_info);
-  addIntToStream("currentConcurrentExecutionCount", 0,          cluster_info);
-  addIntToStream("latencyExecute_mean",             0,          cluster_info);
+  addIntToStream("rollingCountShortCircuited", 0, cluster_info);
+  addIntToStream("rollingCountSuccess", success, cluster_info);
+  addIntToStream("rollingCountThreadPoolRejected", 0, cluster_info);
+  addIntToStream("rollingCountTimeout", timeouts, cluster_info);
+  addIntToStream("rollingCountBadRequests", 0, cluster_info);
+  addIntToStream("currentConcurrentExecutionCount", 0, cluster_info);
+  addIntToStream("latencyExecute_mean", 0, cluster_info);
 
   // latency information can be  taken rom hystogram, which is only available to sinks
```

Comment:
Few spelling issues:
two spaces before taken, from (and not rom), histogram (and not hystogram)
Also the second line might be a TODO

Reply:
i already fixed it. wonder if this change was already lost :(